packages/neuron-wallet/src/services/multisig.ts

```diff
@@ -62,16 +71,219 @@ export default class MultisigService {
       .where({
         walletId
       })
+      .orderBy('id', 'DESC')
       .getMany()
     return result
   }
 
   async deleteConfig(id: number) {
+    const config = await getConnection()
+      .getRepository(MultisigConfig)
+      .createQueryBuilder()
+      .where({
+        id
+      })
+      .getOne()
+    await getConnection().manager.remove(config)
+  }
+
+  static async getLiveCells(multisigConfigs: MultisigConfig[]) {
+    const liveCells: MultisigOutput[] = []
+    const addressCursorMap: Map<string, string> = new Map()
+    let currentMultisigConfigs = multisigConfigs
+    while (currentMultisigConfigs.length) {
+      const res = await rpcBatchRequest(
+        IndexerService.LISTEN_URI,
+        currentMultisigConfigs.map(v => {
+          const script = addressToScript(v.fullPayload)
+          return {
+            method: 'get_cells',
+            params: [
+              {
+                script: {
+                  code_hash: script.codeHash,
+                  hash_type: script.hashType,
+                  args: script.args
+                },
+                script_type: 'lock',
+                filter: {
+                  block_range: v.lastestBlockNumber ? [v.lastestBlockNumber, max64Int] : undefined
+                }
+              },
+              'desc',
+              '0x64',
+              addressCursorMap.get(v.fullPayload)
+            ]
+          }
+        })
+      )
+      const nextMultisigConfigs: MultisigConfig[] = []
+      res.forEach((v, idx) => {
+        if (!v.error && v?.result?.objects?.length) {
+          addressCursorMap.set(currentMultisigConfigs[idx].fullPayload, v?.result?.last_cursor)
+          liveCells.push(...v.result.objects.map((output: any) => MultisigOutput.fromIndexer(output)))
```

Comment:
cells with `type script` should be eliminated

Reply:
https://github.com/nervosnetwork/neuron/pull/2340